src/model/Member.ts

```diff
@@ -0,0 +1,7 @@
+export type MemberType = {
+  id: number;
+  name: string;
+  surname: string;
+  profile_image: string;
+  created_at: string;
```

Comment:
I dont think we should use snake_case around the project. I propose two solutions, pick one you like the most

a) we will rename the columns to camelCase
b) we use renaming via select (I believe it should be possible, at least chatgpt think so, I didn't check the docs): 
```
.select(`
      id,
      first_name:firstName,
      last_name:lastName,
      email,
      created_at:createdAt
    `);
```
the problem is, this might lead to typos/mistakes. I suggest going with the first one